### DISCUSSION TITLE
How To Use Post-processing

### DISCUSSION THREAD

**Discussion by khaledOghli** [Tresjs/tres#372] [Q&A] [answered]
Hi ,
How To use Post Processing,
I see this playground but it's not working anymore.
https://playground.tresjs.org/experiments/post-processing-manual
I think there is an update of tres js on how to use Post Processing.!!

**Comment by alvarosabu** [comment, marked as answer]
Hi @khaledOghli did you tried https://github.com/Tresjs/post-processing
